src/components/Person/Person.jsx

```diff
@@ -1 +1,25 @@
-// export const Person = ({ person }) => ();
+const PERSON_MAN_SEX = 'm';
+
+const Person = ({ person }) => {
+  const { name, age, partnerName, sex, isMarried } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
```

Comment:
It is worth converting this value to boolean manually, because if there is 0, it will simply render 0:
```suggestion
      {!!age && (
```